src/tests/test_process.py

```diff
@@ -147,7 +147,7 @@ def _test_threads(self):
                 "ps", "-L", "-o", "comm=", "--pid",
                 str(self.process.pid)
             ]).split()
-        except:  # pylint: disable=bare-except
+        except:  # noqa:E722
```

Comment:
investigate, also error may be different on windows

Reply:
Hmm, you decided to switch from pylint to flake8, why?

Reply:
because pylint ignore comment not able to suppress flake8 warnings.
But flake8 comments is able to suppress both linter warnings.

Reply:
Updated subprocess.CalledProcessError